pkg/cmd/scan/output/html.go

```diff
@@ -81,6 +81,16 @@ func (c *HTML) Write(analysis *analyser.Analysis) error {
 
 			return distinctResourceTypes(resources)
 		},
+		"getIaCSources": func() []string {
```

Comment:
This could be changed to 
```go
		"getIaCSources": func() []string {
			resources := make([]*resource.Resource, 0)
            resources = append(resources, analysis.Deleted()...)
            resources = append(resources, analysis.Managed()...)
			return distinctIaCSources(resources)
		},
```

We may want to filter on IaC source for resource that may not have diffs (thinking of non deep mode which is now the default)

Reply:
We don't display managed resources in html output, but yeah it make sense to display there's no drift for a single IaC source for example